packages/swap/src/routes/quote.ts

```diff
@@ -63,6 +63,13 @@ const quote = (io: Server) => {
 
       logger.info('received a quote request', { query: req.query });
 
+      // detect if ingress and egress fees are exposed as gas asset amount or fee asset amount
+      // https://github.com/chainflip-io/chainflip-backend/pull/4497
+      // TODO: remove this once all networks are upraded to 1.3
+      const ingressEgressFeeIsGasAssetAmount =
```

Comment:
this works because `FLIP` and `USDC` ave a different number of decimals. so if the ingress fee is the same for both of them, we can be sure that it is the old version of the rpc (the value is a gas asset amount, not a FLIP/USDC amount)

Reply:
Nice trick!